nextcord/ext/commands/core.py

```diff
@@ -64,6 +64,7 @@
 
 __all__ = (
     "Command",
+    "Cog",
```

Comment:
why is this re-exported here? cog is in `commands/cog.py`

Reply:
done